docker/openemr/7.0.1/utilities/devtoolsLibrary.source

```diff
@@ -23,6 +23,9 @@ prepareVariables() {
     else
         CUSTOM_DATABASE="openemr"
     fi
+    if [ "$MYSQL_PORT" != "" ]; then
+        CONFIGURATION="${CONFIGURATION} port=${MYSQL_PORT}"
+    fi
```

Comment:
would rec following how is done in the $MYSQL_DATABASE block above where is converted to $CUSTOM_DATABASE and the default port of 3306 is set if not given. and would then use the $CUSTOM_MYSQL_PORT setting below.

Reply:
Updated